postgres-appliance/callback_aws.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+import boto3
+import logging
+import requests
+import sys
+import time
+
+from botocore.exceptions import ClientError
+from requests.exceptions import RequestException
+
+
+logger = logging.getLogger(__name__)
+
+
+def retry(func):
+    def wrapped(*args, **kwargs):
+        count = 0
+        while True:
+            try:
+                return func(*args, **kwargs)
+            except ClientError as e:
+                if e.response['Error']['Code'] != 'Throttling' or 'RequestLimitExceeded' not in str(e):
```

Comment:
this is quite confusing and   is better to be spelled as ``not ( e.response['Error']['Code'] == 'Throttling' or 'RequestlimitExceeded' in str(e))``. Makes it clearer that it should be and in the original condition.